### ISSUE TITLE
SurfaceOM removal is confusing

### ISSUE THREAD

**Issue by hut104** [APSIMInitiative/ApsimX#6614]
People find the current method for removing residues from the soil surface to be very confusing.
To remove residues, one has to till a fraction of the residues to a zero depth.  We should have a more explicit method for removing a fraction of the soil surface residues. This could then be used for burning or removal treatments.

**Comment by mppalves** [comment]
Hi @hut104 Thanks for the comment, I support your statement. 

I'm very new to APSIM and I need some help understanding how I can implement different types of tillage/residue management. Would have some material you could share, or examples of how this could be done?

Thanks!

**Comment by soilmodel12** [comment]
Agreed this is very confusing! @hut104 could this be explained in more detail?

**Comment by soilmodel12** [comment]
@mppalves did you figure this out?

**Comment by mppalves** [comment]
Hi @soilmodel12, not yet...

**Comment by JJguri** [comment]
Hi @hut104, totally agree with you. I am facing the same issue particularly trying to mix different tillage (conventional vs zero tillage) and residue removal treatments (with different percentage of stover removal) in Classic. I imagine the tillage module is the same between Classic and NG. Below a more detail explanation:

Below you can see a simulation with different treatments including wheat (W) and maize (M) monocultures with conventional tillage (CT) and zero tillage or no-till (ZT) and residue removals (R000, no residue removal; R100, 100% residue removal). Residue indicates all stover after grain harvest. I am using the tillage module to do both, the tillage event 3 weeks before sowing and the residue removal after harvest with the user_defined function (.apsim and .met files attached). I found that APSIM is not able to differentiate between CT and ZT (red and orange lines, they overlap) on changes in organic carbon (%) when the treatment includes 100% residue removal due to the tillage event does not affects FOM (due to surface_om after harvest = 0). On the other hand, I am having model under-estimations for the ZT treatment with no residue removal (green line; actual observed values are at least 0.5% higher in oc). Could you please provide your opinion about possible fixes in the model to catch the actual management in these trials?

I have a couple of doubts:

- Am I using the right way to implement the residue removal and tillage events? are there other ways in the model?
- Am I counting for the higher microbial activity in a ZT system vs a CT system?
- Which parameters should I check that affects the changes of organic carbon when these practices (tillage and residue removal) are applied together?

<img width="2046" alt="Screenshot 2023-11-01 at 08 23 41" src="https://github.com/APSIMInitiative/ApsimX/assets/41041705/be08ed92-8403-4ba9-83cc-5305ea1f5598">

[el_batan.zip](https://github.com/APSIMInitiative/ApsimX/files/13249513/el_batan.zip)
[el_batan_met.zip](https://github.com/APSIMInitiative/ApsimX/files/13249514/el_batan_met.zip)
